(/authentication/social-connections/coinbase) update guide (#1644)
clerkinc/clerk-docs: docs/authentication/social-connections/coinbase.mdx

1. In the **Choose provider** dropdown, select **Coinbase**.
1. Select **Add connection**.

## Configure for your production instance

For _production instances_, you must provide custom credentials which involves generating your own **Client ID** and **Client Secret** using your Coinbase account.

To make the setup process easier, it's recommended to keep two browser tabs open: one for your [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) and one for your [Coinbase's Developer Platform](https://portal.cdp.coinbase.com).

<Steps>
  ### Enable Coinbase as a social connection in Clerk

  1. In the Clerk Dashboard, navigate to the [**SSO Connections**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) page.
  1. Select **Add connection** and select **For all users**.
  1. In the **Choose provider** dropdown, select **Coinbase**.
  1. Ensure that both **Enable for sign-up and sign-in** and **Use custom credentials** are toggled on.
  1. Save the **Redirect URI** somewhere secure. Keep the modal and page open.

  ### Create a Coinbase app

  > [!NOTE]

>   > Coinbase automatically creates a default project for you named `Project 1`. Select the icon next to the project name to rename.

Reviewer: ```suggestion
  > Coinbase automatically creates a default project for you named `Project 1`. Select the icon next to the project name to rename it.
```

  1. In your [Coinbase Developer Platform project dashboard](https://portal.cdp.coinbase.com/projects), navigate to the **API Keys** tab.
  1. Select **OAuth**, then select **Create client**.
  1. Complete the required fields. In **Redirect URIs**, paste the **Redirect URI** you saved from the Clerk Dashboard.

>   1. Select **Create client**. The page will refresh and you should see the **Client ID** and **Client Secret** in the **OAuth** section. Save both values somewhere secure. Keep this page open.

Reviewer: they actually never have to come back to this page in the tutorial, so its fine to close it.

```suggestion
  1. Select **Create client**. The page will refresh and you should see the **Client ID** and **Client Secret** in the **OAuth** section. Save both values somewhere secure.
```

  ### Set the Client ID and Client Secret in your Clerk Dashboard

>   1. Navigate back to your Clerk Dashboard where the modal should still be open. Paste these values into the respective fields.

Reviewer: we can't say "these" because we don't specify which values, in this section. 

```suggestion
  1. Navigate back to your Clerk Dashboard where the modal should still be open. Paste the **Client ID** and **Client Secret** values into the respective fields.
```

>   1. Navigate back to your Clerk Dashboard where the modal should still be open. Paste the **Client ID** and **Client Secret** values into the respective fields.

Reviewer: ```suggestion
  1. Navigate back to your Clerk Dashboard where the modal should still be open. Paste the **Client ID** and **Client Secret** values that you saved into the respective fields.
```

  1. Select **Add connection**.

  > [!NOTE]

>   > If the modal or page is not still open, navigate to the [**SSO Connections**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) page in the Clerk Dashboard. Select the Coinbase connection. Under **Use custom credentials**, you can paste the **Client ID** and **Client Secret** into their respective fields.

Reviewer: ```suggestion
  > If the modal or page is not still open, navigate to the [**SSO Connections**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) page in the Clerk Dashboard. Select the **Coinbase** connection. Under **Use custom credentials**, you can paste the **Client ID** and **Client Secret** into their respective fields.
```
